src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,59 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's block in the address book.
+ * Guarantees: immutable;  is valid as declared in {@link #isValidBlock(String)}
+ */
+public class Block {
+
+    public static final String EXAMPLE = "123";
+    public static final String MESSAGE_BLOCK_CONSTRAINTS = "Person block can be in any format";
```

Comment:
If the `MESSAGE_BLOCK_CONSTRAINT` string is not used anywhere outside of the class, do change the access modifier.